src/app/venter/complaint/detailed-complaint.component.css

```diff
@@ -0,0 +1,180 @@
+.container {
+    margin: 0 175px;
+    padding: 0 5%;
+}
+
+.placeholder-image {
+    background-size: cover;
+}
+
+/* Slideshow container */
+
+.slideshow-container {
+    max-width: 500px;
+    max-height: 500px;
+    position: relative;
+    margin: auto;
+}
+
+.display-image {
+    max-height: 400px;
+    max-width: 400px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/wncc/instiapp-angular/pullRequest?prid=5455571)